### PR TITLE
chore: cleaning up sentry logs

### PR DESCRIPTION
**Pull request by y-lohse**
The commit titles explain al that's going on :)
Once we're good to merge this PR, I'll purge the existing logs in sentry and we should be good to go!

### CODE REVIEW

**Comment by y-lohse** [comment]
[(1) 📱 Mettre Sentry au propre](https://trello.com/c/UrpdJUeD/508-1-%F0%9F%93%B1-mettre-sentry-au-propre)

File: config/webpack.target.mobile.js [hunk 1]
```diff
@@ -20,7 +20,6 @@ module.exports = function (production, app) {
         __ALLOW_HTTP__: !production,
         __TARGET__: JSON.stringify('mobile'),
         __SENTRY_TOKEN__: JSON.stringify('29bd1255b6d544a1b65435a634c9ff67'),
-        __DEVMODE__: !production,
```
Comment (enguerran) [inline config/webpack.target.mobile.js]: cannot see the point with sentry

File: src/drive/mobile/actions/settings.js [hunk 2]
```diff
@@ -32,7 +31,6 @@ export const setAnalytics = (analytics, source = 'settings') => (dispatch, getSt
   if (analytics && state.mobile) {
     const value = state.mobile.settings.backupImages
     logInfo(`${source}: backup images is ${value ? 'enabled' : 'disabled'}`)
-    dispatch(pingOnceADay(state.mobile.timestamp, analytics))
```
Comment (enguerran) [inline src/drive/mobile/actions/settings.js]: Why did you remove the `pingOnceADay` feature?

**Comment by y-lohse** [comment]
The story is linked in the issue, but it's [this one](https://trello.com/c/UrpdJUeD/508-1-%F0%9F%93%B1-mettre-sentry-au-propre) ;). And so yes, if I understand correctly we want to remove usage data — probably in favor of piwik.

File: config/webpack.target.mobile.js (hunk 1 above)
Comment (y-lohse) [reply config/webpack.target.mobile.js]: I'm not sure I understand what you mean here.

Comment (enguerran) [reply config/webpack.target.mobile.js]: You create a PR named `chore: cleaning up sentry logs` and I saw something not related.
I guess it is easier for you to put the whole work day in a simple Pull Request, but as I wrote earlier on #398 it is less mess to do PR related to one and only one consideration. Let's group our work by topic instead of `git commit -am 'my work for the day' &&  git push && git pull-request -m 'please sort my work for me and review'`.

Comment (y-lohse) [reply config/webpack.target.mobile.js]: Ah I see, but I wouldn't say that's the case here :). If I didn't remove the `__DEVMODE__` flag in this PR, I would be using both `__DEVLOPMENT__` and `__DEVMODE__` inside the same file (`reporter.js`) — and that smells. So I did some light refactoring along the way, as we said we'd do :)

Comment (enguerran) [reply config/webpack.target.mobile.js]: Maybe its Friday and I lose all the patience of mine in #398

Comment (y-lohse) [reply config/webpack.target.mobile.js]: No problem ;). I'm still looking for my courage before diving into it.
